cmake/DPF-plugin.cmake

```diff
@@ -366,7 +366,12 @@ function(dpf__add_dgl_cairo)
   target_compile_definitions(dgl-cairo-definitions INTERFACE "DGL_CAIRO" "HAVE_CAIRO")
 
   target_include_directories(dgl-cairo PUBLIC ${CAIRO_INCLUDE_DIRS})
-  target_link_libraries(dgl-cairo PRIVATE dgl-cairo-definitions ${CAIRO_LIBRARIES})
+  if(NOT MINGW)
```

Comment:
can you reverse this condition? if mingw first, much easier to understand and matches the changes below.
thanks